v1/tester/fixture_test.go

```diff
@@ -5,16 +5,25 @@ const fixtureReporterVerboseBenchmark = `FAILURES
 data.foo.bar.test_corge: FAIL (0s)
 
 
+data.foo.bar.test_cases_fail: FAIL (0s)
+
+
+  two: FAIL
```

Comment:
The double newlines here look a little sus

Reply:
I don't think this will actually ever happen outside of testing. This test has verbose mode (`-v`) turned on, which will include trace output between those newlines. The mocked result set doesn't include traces, but for proper runs, it'll always be there. The `data.foo.bar.test_corge: FAIL (0s)` output above this has the same apparent issue.
I'll add some mocked tracing to make the output more resemble the real thing.